package.json

```diff
@@ -29,22 +30,23 @@
     "ember-cli-qunit": "0.3.13",
     "ember-cli-uglify": "^1.0.1",
     "ember-data": "1.13.9",
+    "ember-disable-prototype-extensions": "^1.0.0",
     "ember-disable-proxy-controllers": "^1.0.0",
     "ember-export-application-global": "^1.0.2",
-    "ember-disable-prototype-extensions": "^1.0.0",
     "ember-try": "0.0.6",
     "ember-watson": "^0.6.2"
   },
   "keywords": [
     "ember-addon"
   ],
   "dependencies": {
-    "ember-bootstrap-controls": "git+https://github.com/wildland/ember-bootstrap-controls#v0.0.8",
+    "active-model-adapter": "2.1.1",
+    "ember-bootstrap-controls": "git+https://github.com/wildland/ember-bootstrap-controls#ember-update",
```

Comment:
This will need to change before merging. Therefore we need to merge and set the version of ember bootstrap controls that this pulls from.